src/docs/organization.yaml

```diff
@@ -3,6 +3,67 @@ tags:
       description: API to manage the organization account.
 
 paths:
+    /api/organization/{id}/subscription:
```

Comment:
replace this with `subscription` with `sub`, because the delete endpoint was implemented this way.